update-antora-spring-ui/action.yml

```diff
@@ -0,0 +1,88 @@
+name: Update Antora UI Spring
+description: A GitHub Action that detects if there is a new version of the Antora UI Spring artifact and creates a PR to update the antora-playbook.yml file to use the new version.
+
+inputs:
+  docs-branch:
+    description: The branch where the antora-playbook.yml file is. Default is docs-build
+    default: 'docs-build'
+  workflow-branch:
+    description: The branch that will be created by the workflow with the changes. Default is update-antora-ui-spring
+    default: 'update-antora-ui-spring'
+  token:
+    description: Token with write permission to pull-requests, issues and contents
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v4
+    - id: check-existing-pr
+      name: Check for Existing PR
+      run: |
+        pr_count=$(gh pr list --head ${{ inputs.workflow-branch }} --base ${{ inputs.docs-branch }} --state open --json id | jq length)
+        if [[ $pr_count -eq 0 ]]; then
+          echo "continue=true" >> $GITHUB_OUTPUT
```

Comment:
Do we want it to stop if there is an existing PR? What happens if a PR is created for version 0.0.1 and then a newer version of the UI becomes available? Should we perhaps update the existing PR in that case?

At minimum, I think we should document (in the README) what happens when a PR already exists. We should also document how to get it to create a PR for the latest version (e.g. close the PR and delete the branch). It would be valuable to echo out that it is not continuing looking for a newer version and why with a link to the section of the README too.

Reply:
I've added to the README what happens when there is an open PR